aspnet/signalr/overview/security/introduction-to-security.md

```diff
@@ -80,6 +80,13 @@ For each request, the server validates the contents of the token to ensure that
 
 Because the connection id is part of the verification process, you should not reveal one user's connection id to other users or store the value on the client, such as in a cookie.
 
+### Note: Connection Tokens vs Other Types of Tokens
+Connection tokens are occasionally flagged by security tools because they appear to potentially be session tokens or authentication tokens, which would pose a risk if exposed.
+
+In SignalR, the The Connection Token is not an authentication token. It is used to confirm that the user making this request is the same one that created the connection. This is needed because ASP.NET SignalR allows connections to move between servers. The connection token contains the "connection ID" and the user name of the user that created the connection, and is encrypted with a key known only to the servers. The server that initialized the connection checks the user ID provided by your authentication mechanism and embeds it in this token so that if a request is made to a different server, it can verify that the connection ID the client presents is valid and was created on behalf of this user. However the token itself makes no claim that the request was made by that user, only that the connection ID contained within the token is associated with that user.
```

Comment:
the The Connection Token

->

the connection token

--

extra "the", and connection token isn't a proper noun here, so we don't need Title Case.

Reply:
This is needed because ASP.NET SignalR allows connections to move between servers.

->

The connection token is necessary because ASP.NET SignalR allows connections to move between servers.

Reply:
> The connection token contains the "connection ID" and the user name of the user that created the connection, and is encrypted with a key known only to the servers. The server that initialized the connection checks the user ID provided by your authentication mechanism and embeds it in this token so that if a request is made to a different server, it can verify that the connection ID the client presents is valid and was created on behalf of this user.

I think we can actually simplify this a little:

The token associates the connection with a particular user but doesn't assert the identity of the user making the request. For a SignalR request to be properly authenticated, it must have some other token (Cookie, Bearer Token, etc.) that asserts the identity of the user.

Reply:
Could drop "In SignalR, " at start of paragraph

"This is needed because ASP.NET Core SignalR ..." (add "Core")

etc. is not allowed by our style guide -- one possible way to rephrase: "... some other token that asserts the identity of the user, such as a cookie or bearer token."

Reply:
This isn't a doc for ASP.NET Core SignalR, so don't add "Core"! :)